src/utils/logger.ts

```diff
@@ -27,10 +27,19 @@ logger.add(elasticsearch);
  * @param severity - the kind of log created.
  * @param name - name of the log. in our case, the function called.
  * @param description - description in text.
- * @param correlationId - id to correlate to if there are several logs with some connection.
+ * @param traceID - id to correlate to if there are several logs with some connection.
  * @param user - the user requesting for the service.
  * @param more - additional optional information.
  */
-export const log = (severity: string, name: string, description: string, correlationId?: string, user?: string, more?: any) => {
-  logger.log(severity, { name, description, correlationId, user, ...more });
+export const log = (severity: Severity, name: string, description: string, traceID?: string, user?: string, more?: any) => {
```

Comment:
Think about what happens when `more` is a string...

Reply:
And other cases.

Reply:
Rename `more` to `meta`.

Reply:
done.